saleor/graphql/core/utils/__init__.py

```diff
@@ -132,3 +136,9 @@ def ext_ref_to_global_id_or_error(model, external_reference):
             message=f"Couldn't resolve to a node: {external_reference}",
             code="not_found",
         )
+
+def cut_file_name_to_max_length(file_name: str) -> str:
+    return (
+        file_name[:FILE_NAME_MAX_LENGTH]
+        if len(file_name) > FILE_NAME_MAX_LENGTH else file_name
```

Comment:
The `if` is not necessary; if you do `[:FILE_NAME_MAX_LENGTH]` on a shorter string, it will leave it intact. Then we wouldn't need this function and could do that in the `get_filename_from_url` directly.

Reply:
yup, I remove function.